### ISSUE TITLE
Feature Request - Add wildcards for the settings.

### ISSUE THREAD

**Issue by hacksenwerk** [pmarchwiak/auto-reader-view#10]
Hi. Nice extension. I tried just a simple 
`*`
in the settings, to enable the reader view by defult on all websites, but it doesn't work.

**Comment by ololiuhqui** [comment]
> Hi. Nice extension. I tried just a simple `*` in the settings, to enable the reader view by defult on all websites, but it doesn't work.

Yes that would be great, then the toggle would deactivate it for the current website "whitelisting" it.
